app/frontend/actions.ts

```diff
@@ -100,7 +108,7 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
     }
   }
 
-  const setAuthMethod = (state, option) => {
+  const setAuthMethod = (state, option: AuthMethodEnum): void => {
```

Comment:
Since we are already modifying this, could we type the state as well?

Reply:
Sure, we can type it bit by bit with every PR